content/en/docs/features/monitoring-and-diagnostics/monitoring-device-connectivity.md

```diff
@@ -0,0 +1,22 @@
+---
+title: 'Monitoring Device Connectivity'
+description: 'Understanding how to determine if a device is online or offline'
+date: '2023-08-07'
+categories: [features]
+keywords: ['service']
+weight: 2
+---
+
+When devices are successfully connected, authenticated, and authorized, the CoAP gateway initiates an `UpdateDeviceMetadata` process within the ResourceAggregate. This process sets the device's connection state to 'Online' and online validity determined by JWT expiration. Conversely, when a device becomes disconnected or the CoAP gateway is gracefully shut down (e.g., using SIGTERM), an event is sent indicating that the device has gone offline. However, there are scenarios to consider, such as when the CoAP gateway terminates unexpectedly due to reasons like being killed for exceeding memory limits in a Kubernetes (k8s) environment or due to power loss.
```

Comment:
`and online validity determined by JWT expiration.`
->
`and the online validity is determined by JWT expiration.`

Reply:
fixed